x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_fetch_alerts.tsx

```diff
@@ -266,11 +266,11 @@ const useFetchAlerts = ({
                   searchSubscription$.current.unsubscribe();
                 }
               },
-              error: (msg) => {
+              error: (error) => {
                 dispatch({ type: 'loading', loading: false });
                 onLoaded?.();
-                data.search.showError(msg);
                 searchSubscription$.current.unsubscribe();
+                throw error;
```

Comment:
If we don't re-throw the error it will be caught in the Observable chain and thus won't surface up to the boundary. Errors will be eventually caught there (and since the useFetchAlerts hook is being used only in the AlertsTable this should be safe enough) but if you prefer to catch some of them locally I can revert this

Reply:
We 100% should avoid crashing if possible. If we know what the error is, we should recover it if we can.